db/migrate/20150419175648_menu_items_orders.rb

```diff
@@ -0,0 +1,8 @@
+class MenuItemsOrders < ActiveRecord::Migration
+  def change
+    create_table :menu_items_orders do |t|
+      t.belongs_to :order, index: true
+      t.belongs_to :menu_item, index: true
+    end
+  end
+end
```

Comment:
We do not need specs for this table as this is used only by Rails to implement the HABTM relationship. We should not be testing that.